src/js/Card.js

```diff
@@ -0,0 +1,32 @@
+import { createElement } from './library.js';
+
+export default class Card {
+  constructor(character) {
+    this.container = '';
+    this.character = character;
+    return this.container;
+  }
+
+  set character(character) {
+    this.container = createElement('div', 'flex-col card');
```

Comment:
- [ ] OPTIONAL: I think card elements can be wrapped in an `article` tag, which is more semantic. I suggest you fix this for best practice's sake. It is not mandatory for your project's submission though.